### ISSUE TITLE
rendering proc gen results

### ISSUE THREAD

**Issue by bjuergens** [neuroevolution-ai/NeuroEvolution-CTRNN_new#40]
related/blocked by #38

problem: our current rending code doesn't work for procgen, because their were some changes

possible solution: https://discourse.aicrowd.com/t/how-to-save-rollout-video-render/3246/10

possible bonus: We can render on the server. 

possible followup issue: life rendering of procgen may require implementing a custom render method,

**Comment by bjuergens** [comment]
implemented in #38

**Comment by bjuergens** [comment]
closed because implemented
